.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,7 @@
 - [ ] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
 - [ ] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
 - [ ] I've updated the documentation if necessary.
+- [ ] I've run `bundle exec fastlane generate_swift_api` if necessary.
```

Comment:
You do not need to run `generate_swift_api`. This is done when new versions of fastlane are made. Can we remove this? 😊